torchgeo/datasets/glacier_mapping_alps.py

```diff
@@ -0,0 +1,427 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""GlacierMappingAlps Dataset."""
+
+import os
+from collections.abc import Callable, Sequence
+from pathlib import Path
+from typing import ClassVar
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import torch
+from matplotlib.figure import Figure
+from torch import Tensor
+
+from .errors import DatasetNotFoundError
+from .geo import NonGeoDataset
+from .utils import (
+    download_and_extract_archive,
+    download_url,
+    extract_archive,
+    lazy_import,
+)
+
+
+class GlacierMappingAlps(NonGeoDataset):
+    r"""A Multi-modal Dataset for Glacier Mapping (Segmentation) in the European Alps.
+
+    The dataset consists of Sentinel-2 images from 2015 (mainly), 2016 and 2017, and binary segmentation masks for
```

Comment:
Would be nice if we could limit all lines to 88 chars, but if ruff doesn't complain it isn't technically required